### DISCUSSION TITLE
Lavalink music bot stop playing and disconnecting in random moment

### DISCUSSION THREAD

**Discussion by TheLisdan** [lavalink-devs/Lavalink#856] [Q&A] [answered]
In random moment music just stop playing with this exception in console
![изображение_2023-03-23_112319231](https://user-images.githubusercontent.com/99090966/227158877-b375ccaf-ddc0-4ef0-b544-009f3127c0dd.png)
![изображение_2023-03-23_112337927](https://user-images.githubusercontent.com/99090966/227158955-2e16023c-66e1-440a-b858-28f0464b52da.png)

**Comment by topi314** [comment, marked as answer]
connection timeouts aren't a thing lavalink can fix for you

that is either a problem with google or your internet connection

**Reply by TheLisdan** [reply]
So, how i can fix it?

**Reply by TheLisdan** [reply]
My track playing normally, but in random moment just stops

**Reply by topi314** [reply]
there isn't an easy solution for this, it could be anything
it's the internet after all, connection issues happen all the time

You only need to worry about it, if it happens more than randomly
